app/backend/src/clj/teet/contract/contract_db.clj

```diff
@@ -119,6 +119,13 @@
   [[contract status]]
   (assoc contract :thk.contract/status status))
 
+(defn contract-with-status-and-manager
+  "Used after datomic query which returns [contract contract-status manager]"
+  [[contract status manager]]
```

Comment:
the manager is not needed here anymore since the manager is fetched in the pull pattern of the contract.